shared-libs/message-utils/src/index.js

```diff
@@ -95,6 +112,12 @@ const getRecipient = function(context, recipient) {
     const type = recipient.split(':')[1];
     phone = getParentPhone(context.patient, type) ||
             getParentPhone(context, type);
+  } else if (recipient.startsWith('contact:')) {
+    const tag = recipient.split(':')[1];
+    phone = getLinkedPhone(context.patient, tag) ||
+            getLinkedPhone(context.contact, tag) ||
+            getParentPhone(context.patient, tag) ||
+            getParentPhone(context.contact, tag);
```

Comment:
This line is ever so slightly different from line 114 where we just pass in the `context`. Is this intentional?

Reply:
Yes! Because I don't want `parent` to be resolved to "clinic", I want parent to be resolved to `contact.parent` and https://github.com/medic/cht-core/blob/master/shared-libs/message-utils/src/index.js#L162

Reply:
Ahh right! Thanks.